app/assets/javascripts/shared/time/timeDirective.spec.js

```diff
@@ -39,11 +62,13 @@ describe('ocTime', function () {
       expect(directiveElem.text()).toBe('02:03');
     });
 
+/*
     it('should update time', function () {
-      scope.mySeconds = 200;
-      scope.$digest();
+      // tähän joku millä saa kasvatettua nykyistä aikaa
```

Comment:
This test shoudn't test it updates automatically because the next test does that. Only when variable `myDate` changes on `scope` should be tested similar to in the older version. So no mocking is actually needed :smile: